scripts/build.ps1

```diff
@@ -95,8 +95,8 @@ param (
     [string]$Config = "Debug",
 
     [Parameter(Mandatory = $false)]
-    [ValidateSet("x86", "x64", "arm", "arm64")]
-    [string]$Arch = "x64",
+    [ValidateSet("x86", "x64", "arm", "arm64", "current")]
+    [string]$Arch = "current",
```

Comment:
I'd prefer to not add support for this feature. Why is it necessary?

Reply:
Because we want to be able to support natively building on arm64 platforms. If we hardcode x86_64, that won't "Just work", and will require explicit configuration.

Reply:
I don't understand why passing `arm64` in for that doesn't work.

Reply:
It does, but what happens when you're building on an arm64 mac. You won't get binaries that run on your current platform. Users shouldn't have to pass something in to get a build for their current platform.

Reply:
For now, I prefer to continue with the current model. We can discuss changing the default behavior of build to "current" separately from this PR.

Reply:
No, this is the PR to do it in, because this is actually where the problem is going to start. We can't just punt things off, that is just saying it will never get done.

Reply:
Let me see if that works with the validate set.

Reply:
Ah that does work. I didn't realized validate set worked with an empty string, but it does, since we do it with Tls. I like empty better.